python/ray/experimental/state.py

```diff
@@ -336,7 +339,7 @@ def log_files(self):
 
     return ip_filename_file
 
-  def task_profiles(self):
+  def task_profiles(self, start=None, end=None, num=None):
```

Comment:
can you document the meaning of these arguments in the docstring? also, do we definitely want to expose the `num` argument here?